sycl/test/xocc_tests/sdaccel_ports/vision/edge_detection/edge_detection_with_webcam.cpp

```diff
@@ -6,7 +6,7 @@
 #include <string>
 
 #include <CL/sycl.hpp>
-#include "device_selectors.hpp"
+#include "../../../utilities/device_selectors.hpp"
```

Comment:
Curious change...
And why duplicating the file into `sycl/test/xocc_tests/sdaccel_ports/vision/edge_detection/device_selectors.hpp` ?